nsv13/code/modules/jobs/security/martial_art.dm

```diff
@@ -100,6 +127,9 @@
 	return FALSE
 
 /datum/martial_art/jujitsu/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	var/def_check = D.getarmor(BODY_ZONE_CHEST, "melee")
+	D.apply_damage(rand(8, 14), STAMINA, blocked = def_check) // stamina damage on harm to safely keep a knocked down person, on the ground
+	D.adjustBruteLoss(rand(-5, -7)) // reduces brute by 60-100% at random
```

Comment:
You realize this can be used to heal people by attacking them right?

Reply:
attacks consistently deal 7 damage, so how would it?